cmd/jira/jira.go

```diff
@@ -289,7 +325,7 @@ func (j *DSJira) EnrichComments(ctx *shared.Ctx, comments []interface{}, item ma
 	for _, comment := range comments {
 		richComment := make(map[string]interface{})
 		for _, field := range shared.RawFields {
-			v, _ := item[field]
+			v := item[field]
```

Comment:
why removing `, _` part? That can cause panics.

Reply:
Static compiler checks, gave me this suggestion.

Reply:
okay, will revert it.

Reply:
done!